### ISSUE TITLE
footnotes a problem?

### ISSUE THREAD

**Issue by davidwhogg** [AASJournals/AASTeX60#59]
I can put in long footnotes, or many footnotes, and they go below the bottom of the page; they never break onto another page or cut the text differently. This is a low-priority issue, because I can do work-arounds, but probably signals some long-term issues with the footnotes or pagination. Let me know if you want a demo.

**Comment by augustfly** [comment]
@davidwhogg yes, a demo would be useful, both of the current and your expected behavior. we are investigating some other footnote issues (respecting margins, rules for crazy intraline spacing) and demos were very helpful there.
